src/index.ts

```diff
@@ -3,55 +3,38 @@
 import { sanitizeInput, getCentury } from "./utils";
 import {
   evaluate,
-  getDefaultOptions,
   isCompany as isCompanyType,
   isPerson as isPersonType,
   isTemporary as isTemporaryType,
-  isTestPerson as isTestPersonType,
   isValidDate,
 } from "./validation";
 import {
   generatePerson,
   generateCompany,
   generateTemporary,
 } from "./generation";
-import { KennitalaInfo, ValidationOptions } from "./types";
+import { KennitalaInfo } from "./types";
 
-export const isValid = (
-  kennitala: string,
-  options?: ValidationOptions
```

Comment:
This is a breaking change to the TypeScript definition. Wonder if we should just deprecate and ignore `options` or bump to v4?

Reply:
Yeah might be good to patch v3 since it's been available for a while now. Will change this to a patch vs major and v4 can be a separate PR

Reply:
Opened another pr #45 

Will publish 3.0.4 with the Misc suggestions PR and that one.